gtk4/src/subclass/editable.rs

```diff
@@ -354,3 +452,21 @@ unsafe extern "C" fn editable_set_selection_bounds<T: EditableImpl>(
         end_position,
     )
 }
+
+pub unsafe trait EditableClassSubclassExt: ClassStruct {
+    #[doc(alias = "gtk_editable_install_properties")]
+    fn install_properties(&mut self) -> u32 {
+        unsafe {
+            let object_class = self as *mut _ as *mut glib::gobject_ffi::GObjectClass;
+
+            let mut first_prop = std::mem::MaybeUninit::uninit();
+            glib::gobject_ffi::g_object_class_list_properties(object_class, first_prop.as_mut_ptr());
+
+            ffi::gtk_editable_install_properties(object_class, 1)
```

Comment:
This is just temporarily, I tried using the reported list length from `list_properties` but i switched to 1 for testing purposes as it's doesn't work as it should for now